src/components/dialogs/network-modifications/voltage-init-modification/voltage-init-modification-dialog.tsx

```diff
@@ -57,6 +57,10 @@ interface CloseFunction {
     (): void;
 }
 
+interface PreviewModeSubmitFunction {
```

Comment:
IPreviewModeSubmitFunction ?

Reply:
Do we have a convention for interface names ?
I'm not sure we have to do it, so I prefer not to change (otherwise I should rename all interfaces in this file).

Reply:
okey